alibi_detect/cd/classifier.py

```diff
@@ -147,12 +148,12 @@ def __init__(
         self._set_config(locals())
 
         backend = backend.lower()
-        if (backend == 'tensorflow' and not has_tensorflow) or (backend == 'pytorch' and not has_pytorch) or \
-                (backend == 'sklearn' and not has_sklearn):
-            raise ImportError(f'{backend} not installed. Cannot initialize and run the '
-                              f'ClassifierDrift detector with {backend} backend.')
-        elif backend not in ['tensorflow', 'pytorch', 'sklearn']:
-            raise NotImplementedError(f"{backend} not implemented. Use 'tensorflow', 'pytorch' or 'sklearn' instead.")
+        BackendValidator(
+            backend_options={'tensorflow': ['tensorflow'],
+                             'pytorch': ['pytorch'],
+                             'sklearn': ['sklearn']},
+            construct_name='ClassifierDrift'
```

Comment:
Could we get the class name dynamically here? Concerned that due to spelling mistakes the error message might be incorrect (although I think it should still validate the backends correctly).